app/models/crime_application.rb

```diff
@@ -1,9 +1,14 @@
 require 'laa_crime_schemas'
 
+# rubocop:disable Metrics/ClassLength
 class CrimeApplication < LaaCrimeSchemas::Structs::CrimeApplication
   include Assignable
   include Reviewable
 
+  def supporting_evidence
+    super.map { |d| Document.new(d.attributes) }
+  end
```

Comment:
Showing my ignorance here - but is `super` a whole list of crime applications then?

Reply:
A list of supporting evidence - d stands for documents